unison-cli/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -349,8 +351,9 @@ isFailure o = case o of
   ListNamespaceDependencies {} -> False
   TermMissingType {} -> True
   DumpUnisonFileHashes _ x y z -> x == mempty && y == mempty && z == mempty
-  NamespaceEmpty _ -> False
-  RefusedToPush{} -> True
+  NamespaceEmpty {} -> True
```

Comment:
While I was here I flipped this to `True` (meaning the "empty namespace" response is a failure) since it seemed more correct. Also updated a transcript.